pkg/api/genqlient.graphql

```diff
@@ -99,6 +105,124 @@ query getDeploymentById($organizationId: ID!, $id: ID!) {
 }
 
 
+# ENVIRONMENTS
+
+query getEnvironmentById($organizationId: ID!, $id: ID!) {
+  environment(id: $id, organizationId: $organizationId) {
+    id
+    name
+    slug
+    description
+    cost {
+      monthly {
+        average {
+          amount
+        }
+      }
+      daily {
+        average {
+          amount
+        }
+      }
+    }
+    packages {
+      id
+      params
+      artifacts {
+        id
+        name
+        field
+      }
+      remoteReferences {
+        artifact {
+          id
+          name
+          field
+        }
+      }
+      status
+      manifest {
+        id
+        name
+        slug
+        suffix
+        description
+        bundle {
+          id
+          name
+          version
```

Comment:
I would exclude this field, it won't be here anymore after versions. We added it just to visually see that OCI was working, we now have a `versions` field that returns all versions.

Reply:
The actual `version` in use will be on the package.

Reply:
Done